Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -352,6 +352,7 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.9,
 )
 
+<<<<<<< HEAD
```

Comment:
looks like leftovers from a merge got included in the commit, these should be removed before we can test.